src/main/java/pl/rpg/menu/Main.java

```diff
@@ -0,0 +1,36 @@
+package pl.rpg.menu;
+
+import pl.rpg.game.Game;
+
+import java.util.Scanner;
+
+public class Main {
+
+  public static void main(String[] args) {
+
+    Scanner scanner = new Scanner(System.in);
```

Comment:
I think that reading from user is additional bussines logic and to be consistent with SOLID(S) You should create additional class or method for this.